src/main/java/ch/njol/skript/expressions/ExprHealth.java

```diff
@@ -41,7 +41,7 @@
  * @author Peter Güttinger
  */
 @Name("Health")
-@Description("The health of a creature, e.g. a player, mob, villager, etc. from 0 to the creature's max health, e.g. 10 for players.")
+@Description("The health of a creature, e.g. a player, mob, villager, etc. From 0 to the creature's max health (e.g. 10 for players).")
```

Comment:
I don't really like the wording here, maybe something like `The minimum value is 0, and the maximum is the creature's max health`?